cmd/plan.go

```diff
@@ -139,10 +140,13 @@ func planHandler(cmd *cobra.Command, flags planFlags) {
 		logrus.Errorf("Unable to write plan file (%s) : %s", planfile, err)
 		return
 	}
+	logrus.Debugf("Plan : %+v", p)
 	logrus.Infof("Plan can be found at [%s].", planfile)
 	if len(p.Spec.Services) == 0 && len(p.Spec.InvokedByDefaultTransformers) == 0 {
-		logrus.Debugf("Plan : %+v", p)
-		logrus.Fatalf("failed to find any services or default transformers.")
+		if flags.failOnEmptyPlan {
+			logrus.Fatalf("Did not detect any services in the directory %s", srcpath)
```

Comment:
we might also want to mention about default transformers similar to the previous message.

Reply:
Done